src/behavior-considered-undefined.md

```diff
@@ -23,9 +23,11 @@ code.
 </div>
 
 * Data races.
-* Evaluating a [dereference expression] (`*expr`) on a raw pointer that is
-  [dangling] or unaligned, even in [place expression context]
-  (e.g. `addr_of!(*expr)`).
+* Accessing (loading from or storing to) a place that is [dangling] or unaligned.
```

Comment:
The "unaligned" here is doing unexpected amounts of work. Cc https://github.com/rust-lang/opsem-team/issues/11

Reply:
Since it doesn't look like we will quickly get consensus to reduce the alignment requirement here, I have added an explanation to the PR:

> The alignment requirement of a place is computed based on the entire place expression since the last `*` (if any).
> If the place was computed via `(*ptr).field.field`, then the final place is considered aligned only if `ptr` is aligned *according to its type*.

This leaves us the option to keep rejecting all the examples in https://github.com/rust-lang/opsem-team/issues/11.